pkg/vere/main.c

```diff
@@ -1820,19 +1828,19 @@ _cw_chop(c3_i argc, c3_c* argv[])
     fprintf(stderr, "king: failed to read lif\r\n");
     u3_king_bail();
   }
-  ver_n = u3i_bytes(ver.mv_size, ver.mv_data);
-  ver.mv_data = &ver_n;
-  who_n = u3i_bytes(who.mv_size, who.mv_data);
-  who.mv_data = &who_n;
-  fak_n = u3i_bytes(fak.mv_size, fak.mv_data);
-  fak.mv_data = &fak_n;
-  lif_n = u3i_bytes(lif.mv_size, lif.mv_data);
-  lif.mv_data = &lif_n;
+  c3_d* ver_d = c3_malloc(ver.mv_size);
```

Comment:
Nitpick: when you have operations 1 and 2 to perform on items a,b,c,d, it's usually better to do:
```
1 a; 2 a
1 b; 2 b
1 c; 2 c
1 d; 2 d
```
rather than
```
1 a; 1 b; 1 c; 1 d
2 a; 2 b; 2 c; 2 d
```
because it's easier to pull out `1 then 2` into a function later.